### PR TITLE
SBERDOMA-687 show error in notification & add empty data state

### PR DESCRIPTION
**Pull request by sitozzz**
- Error display with antd notification
- Empty icon when data not delivered from api
Widget with error

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/organization/utils/accessSchema.js [hunk 1]
```diff
@@ -28,6 +28,26 @@ async function checkOrganizationPermission (userId, organizationId, permission)
     return employeeRole[permission] || false
 }
 
+async function checkOrganizationEmployeePermission (userId, organizationId) {
```
Comment (dkoviazin) [inline apps/condo/domains/organization/utils/accessSchema.js:31]: The name will be confusing for developers. You check here that the user belongs to the organization even without roles
and no permissions check.

Comment (pahaz) [reply apps/condo/domains/organization/utils/accessSchema.js:31]: If you just look at the name, then I would ask how it differs from `checkOrganizationPermission`.
It's confusing me too

File: apps/condo/domains/ticket/schema/TicketReportService.test.js [hunk 2]
```diff
@@ -25,16 +21,16 @@ describe('TicketReportService', () => {
         })
 
         it('can not get ticket report with another organization', async () => {
```
Comment (pahaz) [inline apps/condo/domains/ticket/schema/TicketReportService.test.js:23]: please add two tests for two cases: user without role and user with role

File: apps/condo/domains/organization/utils/accessSchema.js (hunk 1 above)
Comment (sitozzz) [reply apps/condo/domains/organization/utils/accessSchema.js]: @pahaz, @dkoviazin I renamed method and remove deletedAt field from access query

File: apps/condo/domains/ticket/schema/TicketReportService.test.js (hunk 2 above)
Comment (sitozzz) [reply apps/condo/domains/ticket/schema/TicketReportService.test.js:45]: Done

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/ticket/schema/TicketReportService.js [hunk 3]
```diff
@@ -45,7 +45,8 @@ const TicketReportService = new GQLCustomSchema('TicketReportService', {
             schema: 'ticketReportWidgetData(data: TicketReportWidgetInput!): TicketReportWidgetOutput',
             resolver: async (parent, args, context, info, extra) => {
                 const { periodType, offset = 0, userOrganizationId } = args.data
-                const hasAccess = await checkOrganizationPermission(context.authedItem.id, userOrganizationId, 'canManageTickets')
+
+                const hasAccess = await checkUserBelongsToOrganization(context.authedItem.id, userOrganizationId)
```
Comment (pahaz) [inline apps/condo/domains/ticket/schema/TicketReportService.js:49]: you should move this logic inside the access. You also should rename the access function name

**Pull request merged**